1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -14,6 +14,12 @@ let salaries = {
 
 // WRITE BELOW THIS LINE
 
+for (let salary in salaries) {
```

Comment:
the code is correct, just remember `for...in` loops through the **keys** of an object. The keys of `salaries` object are the names (ross, monica, rachel), i.e.
```
for (let name in salaries) {
...
```